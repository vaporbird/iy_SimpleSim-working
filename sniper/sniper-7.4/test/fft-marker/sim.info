{'cmdline': ['../../run-sniper',
             '-n',
             '2',
             '-s',
             'markers:verbose:markers',
             '--roi',
             '--',
             './fft',
             '-p',
             '2',
             '-m',
             '6'],
 'git_revision': '',
 'host': 'vanyo',
 'pin_version': 'unknown',
 'rusage': (0.304739,
            0.051571,
            39924,
            0,
            0,
            0,
            7984,
            0,
            0,
            0,
            168,
            0,
            0,
            0,
            2719,
            19),
 'snipercmd': ['bash',
               '-c',
               '/home/vanyo/sniper/sniper-7.4/lib/sniper -c /home/vanyo/sniper/sniper-7.4/config/base.cfg --general/total_cores=2 --general/output_dir=/home/vanyo/sniper/sniper-7.4/test/fft-marker --config=/home/vanyo/sniper/sniper-7.4/config/nehalem.cfg --config=/home/vanyo/sniper/sniper-7.4/config/gainestown.cfg -g --general/magic=true -g --traceinput/stop_with_first_app=true -g --traceinput/restart_apps=false -g --hooks/numscripts=1 -g --hooks/script0name=/home/vanyo/sniper/sniper-7.4/test/fft-marker/sim.scripts.py -g --hooks/script0args= -g --traceinput/stop_with_first_app=false -g --traceinput/enabled=true -g --traceinput/emulate_syscalls=true -g --traceinput/num_apps=1 -g --traceinput/trace_prefix=/tmp/tmpwB_DP1/run_benchmarks'],
 't_elapsed': 3.7581560611724854,
 't_start': 1656738218.981816,
 'tracecmds': [['/home/vanyo/sniper/sniper-7.4/record-trace',
                '-o',
                '/tmp/tmpwB_DP1/run_benchmarks',
                '--roi',
                '-e',
                '1',
                '-s',
                '0',
                '-r',
                '1',
                '--follow',
                '--routine-tracing',
                '--',
                './fft',
                '-p',
                '2',
                '-m',
                '6']],
 'user': 'vanyo',
 'vmem': 42512384L}
