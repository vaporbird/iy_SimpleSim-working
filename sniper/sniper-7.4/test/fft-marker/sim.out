                                     | Core 0     | Core 1    
  Instructions                       |      32499 |      34145
  Cycles                             |     170195 |     170195
  IPC                                |       0.19 |       0.20
  Time (ns)                          |      63983 |      63983
  Idle time (ns)                     |      34789 |      41664
  Idle time (%)                      |      54.4% |      65.1%
Branch predictor stats               |            |           
  num correct                        |       1295 |       1456
  num incorrect                      |        205 |        231
  misprediction rate                 |     13.67% |     13.69%
  mpki                               |       6.31 |       6.77
TLB Summary                          |            |           
  I-TLB                              |            |           
    num accesses                     |       3328 |       3701
    num misses                       |         23 |         34
    miss rate                        |      0.69% |      0.92%
    mpki                             |       0.71 |       1.00
  D-TLB                              |            |           
    num accesses                     |      18484 |      18923
    num misses                       |         22 |         22
    miss rate                        |      0.12% |      0.12%
    mpki                             |       0.68 |       0.64
  L2 TLB                             |            |           
    num accesses                     |         45 |         56
    num misses                       |         45 |         56
    miss rate                        |    100.00% |    100.00%
    mpki                             |       1.38 |       1.64
Cache Summary                        |            |           
  Cache L1-I                         |            |           
    num cache accesses               |       3328 |       3701
    num cache misses                 |        288 |        342
    miss rate                        |      8.65% |      9.24%
    mpki                             |       8.86 |      10.02
  Cache L1-D                         |            |           
    num cache accesses               |      18484 |      18923
    num cache misses                 |        159 |        178
    miss rate                        |      0.86% |      0.94%
    mpki                             |       4.89 |       5.21
  Cache L2                           |            |           
    num cache accesses               |        490 |        545
    num cache misses                 |        447 |        520
    miss rate                        |     91.22% |     95.41%
    mpki                             |      13.75 |      15.23
  Cache L3                           |            |           
    num cache accesses               |        490 |        545
    num cache misses                 |        346 |        338
    miss rate                        |     70.61% |     62.02%
    mpki                             |      10.65 |       9.90
DRAM summary                         |            |           
  num dram accesses                  |        677 |          0
  average dram access latency (ns)   |     104.49 |        inf
  average dram queueing delay        |      51.06 |       0.00
  average dram bandwidth utilization |      8.00% |      0.00%
Coherency Traffic                    |            |           
  num loads from dram                |         73 |         43
  num loads from dram cache          |          0 |          0
  num loads from remote cache        |          0 |          0
